src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/params.py

```diff
@@ -153,3 +153,29 @@ def register_resource_parameter(command, dest, arg_group=None, required=True):
     c.argument('end_time', arg_group=filter_arg_group_name)
     c.argument('caller', arg_group=filter_arg_group_name)
     c.argument('status', arg_group=filter_arg_group_name)
+
+with ParametersContext(command='monitor action-group show') as c:
+    c.register_alias('action_group_name', ('--name', '-n'))
+
+with ParametersContext(command='monitor action-group delete') as c:
+    c.register_alias('action_group_name', ('--name', '-n'))
```

Comment:
Why aren't these simply registered at the `action-group` level to eliminate duplication?  Also, since action groups are resources, this should support the `--ids` parameter.

Reply:
will do.